### ISSUE TITLE
Versioning for kakoune

### ISSUE THREAD

**Issue by jkonecny12** [mawww/kakoune#315]
Hi all,
I think kakoune is  really good editor and as project with this quality and stability I think it deserves versions.
When kakoune will have releases, it could be easy to add it to the main repositories on many distributions. Which will lead to more attention to this project.

What do you think @mawww ?

**Comment by FranzSkuffka** [comment]
[semver](http://semver.org/) seems sensible.

**Comment by jkonecny12** [comment]
IMHO it is exactly what kakoune could use.

**Comment by lenormf** [comment]
The project doesn't have the stablity needed to establish a versioning system yet. Seeing how many issues are open/fixed on a weekly basis these days, there's no need to burden ourselves with keeping track of a version number, it would change too often to be relevant.

**Comment by jubalh** [comment]
I would also like to add it in the repo of my distribution but a version number should be provided then. At least a 0.0.1 would be nice so one can create an initial package for it.

**Comment by rennis250** [comment]
I disagree that it is too early to release a version. Take the Julia Language project for example (julialang.org). The project sees multiple commits a day, has over a thousand issues/pull requests, and is booming in popularity. Yet, it still manages to maintain a clear separation between a stable, versioned release and a bleeding edge release, without causing any confusion for the end user. Kakoune now seems rather stable, so an initial release candidate at leaset would probably be worthwhile.

**Comment by lenormf** [comment]
Just a reminder that versions come with a changelog, it would be a big help for package maintainers to have a list of major changes to handle revisions.

**Comment by bfontaine** [comment]
Anything new on this?

**Comment by lenormf** [comment]
Duplicate of #89.

**Comment by mawww** [comment]
The current plan is to create the first release of Kakoune once the regex-impl branch is merged. If everything goes well, I expect that to happen in the coming weeks.

**Comment by Delapouite** [comment]
https://github.com/mawww/kakoune/releases/tag/v2018.04.13
